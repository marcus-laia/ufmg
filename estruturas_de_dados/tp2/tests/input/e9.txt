#ORDEM
A B C D E F G H  J K L M N O P Q R S T U X Z W Y I V
#TEXTO
O peito do pe do pedro e preto
