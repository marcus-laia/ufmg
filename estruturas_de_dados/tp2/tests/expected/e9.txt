do 2
e 1
o 1
pe 1
pedro 1
peito 1
preto 1
#FIM
